packages/ppx_default/ppx_default.0.1.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Generate default values for your types"
+maintainer: ["ProgramingIsTheFuture"]
```

Comment:
We would prefer (no pressure, though) to have a contact point (such as an email address) listed for the maintainer.